### ISSUE TITLE
 What is the recommended way to integrate Multisig into SubDAO within your framework?

### ISSUE THREAD

**Issue by alikhabazian** [Plopmenz/dao-organization#3]
If I add the multisig plugin to Sub dao, there's a problem. It can easily revoke permissions like getAdminSettings and getSharedAddressSettings, affecting other plugins needed for parent dao to control sub dao properly.

**Comment by Plopmenz** [comment]
Correct, currently the SubDAO permission is only for the ParentDAO to pledge that it views this other DAO as its department, nothing more than that. It is just for frontends / smart contracts to showcase it as such, but the DAOs are completely separated in every other way.

In the next iteration of the project, it is planned to be able to give the ROOT permission to the ParentDAO, to create the situation you are describing, where the SubDAO is limited in altering itself. I hope to finish this somewhere next month. Keeping this issue open for now until the next version is live.

**Comment by alikhabazian** [comment]
If the SubDAO incorporates the Multisig plugin, it would have the capability to execute revoke and grant actions, potentially leading to the removal of other plugins.

**Comment by Plopmenz** [comment]
Currently yes, this is also in line with how the README describes the SubDAO plugin.

In a future version I will add the ability to prevent the SubDAO from revoking and granting actions (so the ParentDAO can control it). This will be either restricting the dao directly (revoking its ROOT permission, if possible) or restricting the execute of the governance plugins (such as Multisig), to prevent execution when a revoke/grant action is present.
